lib/postprocessing/image.js

```diff
@@ -606,6 +616,25 @@ async function imagePostProcess(intermediateRendition, rendition, directories) {
     await write(img, rendition.path);
 }
 
+/**
+ * Runs cwebp to convert images to webp
+ * @param {Rendition} intermediateRendition existing intermediate rendition created by the worker callback
+ * @param {Rendition} rendition final rendition to write to, including instructions
+ */
+async function convertUsingCwebp(intermediateRendition, rendition) {
+    try {
+
+        const command = `cwebp ${intermediateRendition.path} -metadata icc -o ${rendition.path}`;
```

Comment:
need to implement the other options we apply during imagemagick:
```
  convert -background 'rgba(0, 0, 0, 0.0)' test-worker/test/asset-compute/worker/jpegICC-webp/file.jpeg -auto-orient -orient Undefined -define webp:lossless=false -colorspace sRGB -set comment 'Generated by Adobe Asset Compute SDK post-processing.' output.webp
```
and resizing